@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -1101,6 +1102,27 @@ export class MapMeshStandardMaterial extends THREE.MeshStandardMaterial
         return this;
     }
 
+    /** @override */
+    get displacementMap(): THREE.Texture | null {
+        return super.displacementMap === undefined ? null : super.displacementMap;
```

Comment:
`THREE.MeshStandardMaterial` defines `displacementMap` as `THREE.Texture` or `null`, not undefined, but ok it will work anyway. Have you found from THREE.js code that it may be undefined in some scenario?

Reply:
Yes, it's actually undefined by default. Threejs MeshStandardMaterialParameters has this type for the displacementMap:
displacementMap?: Texture | null;

If it's undefined in the parameters passed to the constructor (as it is right now in our code), the displacementMap member in the material is left with an undefined value.

Reply:
Ok.